thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessResource.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.thirdeye.completeness.checker;
```

Comment:
i would prefer we combine. And keep it in the dashboard/resources, so that all resources will stay in a single place